java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -1030,13 +1026,11 @@ public SargApplier(SearchArgument sarg,
                   LOG.info("Skipping ORC PPD - " + e.getMessage() + " on "
                       + predicate);
                 } else {
-                  if (LOG.isWarnEnabled()) {
-                    final String reason = e.getClass().getSimpleName() + " when evaluating predicate." +
-                        " Skipping ORC PPD." +
-                        " Stats: " + stats +
-                        " Predicate: " + predicate;
-                    LOG.warn(reason, e);
-                  }
+                  final String reason = e.getClass().getSimpleName() + " when evaluating predicate." +
+                      " Skipping ORC PPD." +
+                      " Stats: " + stats +
+                      " Predicate: " + predicate;
+                  LOG.warn(reason, e);
```

Comment:
Shall we revert this change?

Reply:
Hello. This change is intentional.  There is never a reason to wrap a `warn` level message in logging guards: `isWarnEnabled`.  The answer will always be "true"

Reply:
Ah, got it. It makes sense.